web-ui/src/components/pulse/Pulse.test.jsx

```diff
@@ -1,18 +1,17 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
```

Comment:
Simplified this snapshot test.

Reply:
See changes to this file as well by @jackkeller on https://github.com/objectcomputing/check-ins/pull/2402